src/main.c

```diff
@@ -304,6 +296,7 @@ int main(void) {
 
     while(1) {
         //set bms_output
+        bms_input.msTicks = msTicks;
```

Comment:
@skoppula because we still want to send periodic CAN messages even in the case of a FORCED_HANG, we would like time to continue to pass, so we added this. LMK if this is a problem.